content/docs/2.7/scalers/datadog.md

```diff
@@ -99,3 +99,11 @@ spec:
     authenticationRef:
       name: keda-trigger-auth-datadog-secret
 ```
+
+## Polling intervals and Datadog rate limiting
+
+[API Datadog endpoints are rate limited](https://docs.datadoghq.com/api/latest/rate-limits/). Depending on the state of the `ScaledObject` there are two different parameters to control how often (per `ScaledObject`) we query Datadog for a metric.
+
+When scaling from 0 to 1, the polling interval is controlled by KEDA, using [the `spec.pollingInterval` parameter in the `ScaledObject` definition](https://keda.sh/docs/2.6/concepts/scaling-deployments/#pollinginterval). For example, if this parameter is set to `60`, KEDA will poll Datadog for a metric value every 60 seconds while the number of replicas is 0.
```

Comment:
What the link here meant to refer to the 2.6 concepts page or 2.7, given that this is a 2.7 scaler page?

Reply:
As I have moved to relative paths, this is no longer needed